boilerplate/fullstack/remix/app/lib/superTokensHelpers.ts

```diff
@@ -92,88 +93,31 @@ export function createPreParsedRequest(request: Request): PreParsedRequest {
 }
 
 export async function getSessionDetails(
-    request: Request,
+    baseRequest: PreParsedRequest | Request,
     options?: VerifySessionOptions,
-    userContext?: Record<string, unknown>
+    userContext?: Record<string, unknown> | UserContext
 ): Promise<{
     session: SessionContainer | undefined;
     hasToken: boolean;
     hasInvalidClaims: boolean;
+    baseResponse?: CollectingResponse;
```

Comment:
this should not be optional, since it's always being returned from this function